### ISSUE TITLE
imapsync issues copying from Gmail to Mailo: "NO Temporary unavailable"

### ISSUE THREAD

**Issue by vagari** [imapsync/imapsync#338]
Hi all, 

Trying to get away from Google, found a potential replacement (mailo.com), but having a problem migrating my mail. 

Using an M1 Mac and a fairly low volume account to test. Initial `--testslive` went just fine. Grabbed the "Darwin" `.sh` file and added my account info for both addresses, with `--justfolders` and those got copied correctly. Thought everything was good, started the script and... all errors. 

It looks like the error is an IMAP one. But I've been struggling to figure out what it means. I originally thought something to do with the `tmpdir` on my end, so I manually changed that. But then noticed the same error in a Postbox log while trying to copy a random message while logged into both accounts on there. 

I can't tell if it's something I'm doing. Or something on the mailo.com end. 

_Edit:_ Whoops, wasn't thinking, attached the log instead of simply pasting. 

```Here is imapsync 2.200 on host MacBook-Pro.localdomain, a darwin system with 0.1/3.1 free GiB of RAM
with Perl 5.30.2 and Mail::IMAPClient 3.43
Transfer started at Saturday 04 June 2022-06-04 08:05:20 -0400 EDT
PID is 12414 my PPID is 12413
Log file is LOG_imapsync/2022_06_04_08_05_20_126_junk@..._junk@....txt ( to change it, use --logfile path ; or use --nolog to turn off logging )
Load is 3.22 2.74 2.51 on 8 cores
Current directory is /Users/jake/Documents/mail-migrate
Real user id is jake (uid 501)
Effective user id is jake (euid 501)
$RCSfile: imapsync,v $ $Revision: 2.200 $ $Date: 2022/04/04 16:05:55 $ 
Command line used, run by /usr/bin/perl5.30:
/opt/homebrew/Cellar/imapsync/2.200/libexec/bin/imapsync --user1 junk@... --password1 MASKED --host2 mail.mailo.com --user2 junk@... --password2 MASKED --tmpdir ./tmp --gmail1
Temp directory is ./tmp ( to change it use --tmpdir dirpath )
kill -QUIT 12414 # special behavior: call to sub catch_exit
kill -TERM 12414 # special behavior: call to sub catch_exit
kill -INT 12414 # special behavior: call to sub catch_reconnect
kill -HUP 12414 # special behavior: call to sub catch_print
kill -USR1 12414 # special behavior: call to sub toggle_sleep
File ./tmp/imapsync.pid does not exist
PID file is ./tmp/imapsync.pid ( to change it, use --pidfile filepath ; to avoid it use --pidfile "" )
Writing my PID 12414 in ./tmp/imapsync.pid
Writing also my logfile name in ./tmp/imapsync.pid : LOG_imapsync/2022_06_04_08_05_20_126_junk@..._junk@....txt
Modules version list ( use --no-modulesversion to turn off printing this Perl modules list ):
Authen::NTLM         1.09
CGI                  4.48
Compress::Zlib       2.084
Crypt::OpenSSL::RSA  0.31
Data::Uniqid         0.12
Digest::HMAC_MD5     1.01
Digest::HMAC_SHA1    1.03
Digest::MD5          2.55
Encode               3.01
Encode::IMAPUTF7     1.05
File::Copy::Recursive 0.45
File::Spec           3.78
Getopt::Long         2.5
HTML::Entities       3.69
IO::Socket           1.40
IO::Socket::INET     1.40
IO::Socket::INET6    2.72
IO::Socket::IP       0.41
IO::Socket::SSL      2.068
IO::Tee              0.65
JSON                 4.03
JSON::WebToken       0.10
LWP                  6.44
MIME::Base64         3.15
Mail::IMAPClient     3.43
Net::Ping            2.71
Net::SSLeay          1.88
Term::ReadKey        2.38
Test::MockObject     1.20200122
Time::HiRes          1.976
URI::Escape          3.31
Unicode::String      2.10

Info: will resync flags for already transferred messages. Use --noresyncflags to not resync flags.
Host2: probing ssl on port 993 ( use --nosslcheck to avoid this ssl probe ) 
Host2: sslcheck detected open ssl port 993 so turning ssl on (use --nossl2 --notls2 to turn off SSL and TLS wizardry)
SSL debug mode level is --debugssl 1 (can be set from 0 meaning no debug to 4 meaning max debug)
Host1: SSL default mode is like --sslargs1 "SSL_verify_mode=0", meaning for host1 SSL_VERIFY_NONE, ie, do not check the server certificate.
Host1: Use --sslargs1 SSL_verify_mode=1 to have SSL_VERIFY_PEER, ie, check the server certificate. of host1
Host2: SSL default mode is like --sslargs2 "SSL_verify_mode=0", meaning for host2 SSL_VERIFY_NONE, ie, do not check the server certificate.
Host2: Use --sslargs2 SSL_verify_mode=1 to have SSL_VERIFY_PEER, ie, check the server certificate. of host2
Info: turned ON syncinternaldates, will set the internal dates (arrival dates) on host2 same as host1.
Host1: will try to use LOGIN authentication on host1
Host2: will try to use LOGIN authentication on host2
Host1: imap connection timeout is 120 seconds
Host2: imap connection timeout is 120 seconds
Host1: imap connection keepalive is on on host1. Use --nokeepalive1 to disable it.
Host2: imap connection keepalive is on on host2. Use --nokeepalive2 to disable it.
Host1: IMAP server [imap.gmail.com] port [993] user [junk@...]
Host2: IMAP server [mail.mailo.com] port [993] user [junk@...]
Host1: connecting and login on host1 [imap.gmail.com] port [993] with user [junk@...]
DEBUG: .../IO/Socket/SSL.pm:1177: global error: Undefined SSL object
DEBUG: .../IO/Socket/SSL.pm:1177: global error: Undefined SSL object
Host1 IP address: 172.253.63.108 Local IP address: 192.168.1.58
Host1 banner: * OK Gimap ready for requests from 73.68.0.116 s25mb96795259qkg
Host1 capability before authentication: IMAP4rev1 UNSELECT IDLE NAMESPACE QUOTA ID XLIST CHILDREN X-GM-EXT-1 XYZZY SASL-IR AUTH=XOAUTH2 AUTH=PLAIN AUTH=PLAIN-CLIENTTOKEN AUTH=OAUTHBEARER AUTH=XOAUTH AUTH
Host1: success login on [imap.gmail.com] with user [junk@...] auth [LOGIN] or [LOGIN]
Host2: connecting and login on host2 [mail.mailo.com] port [993] with user [junk@...]
DEBUG: .../IO/Socket/SSL.pm:1177: global error: Undefined SSL object
DEBUG: .../IO/Socket/SSL.pm:1177: global error: Undefined SSL object
Host2 IP address: 213.182.55.206 Local IP address: 192.168.1.58
Host2 banner: * OK [CAPABILITY IMAP4rev1 ID LITERAL+ AUTH=LOGIN AUTH=PLAIN STARTTLS] IMAP4rev1 MAILO Ready
Host2 capability before authentication: IMAP4rev1 ID LITERAL+ AUTH=LOGIN AUTH=PLAIN STARTTLS AUTH
Host2: mail.mailo.com says it has CAPABILITY for AUTHENTICATE LOGIN
Host2: success login on [mail.mailo.com] with user [junk@...] auth [LOGIN] or [LOGIN]
Host1: state Authenticated
Host2: state Authenticated
Host1 capability once authenticated: IMAP4rev1 UNSELECT IDLE NAMESPACE QUOTA ID XLIST CHILDREN X-GM-EXT-1 UIDPLUS COMPRESS=DEFLATE ENABLE MOVE CONDSTORE ESEARCH UTF8=ACCEPT LIST-EXTENDED LIST-STATUS LITERAL- SPECIAL-USE APPENDLIMIT=35651584 COMPRESS UTF8 APPENDLIMIT
Host2 capability once authenticated: IMAP4rev1 ID LITERAL+ IDLE QUOTA NAMESPACE SORT UIDPLUS UNSELECT
Host1: found ID capability. Sending/receiving ID, presented in raw IMAP for now.
In order to avoid sending/receiving ID, use option --noid
Sending: 4 ID ("name" "imapsync" "version" "2.200" "os" "darwin" "vendor" "Gilles LAMIRAL" "support-url" "https://imapsync.lamiral.info/" "date" "04-Apr-2022 16:05:55 +0000" "side" "host1")
Sent 182 bytes
Read: 	* ID ("name" "GImap" "vendor" "Google, Inc." "support-url" "https://support.google.com/mail" "version" "gmail_imap_220520.08_p0" "remote-host" "73.68.0.116")
  	4 OK Success
Host2: found ID capability. Sending/receiving ID, presented in raw IMAP for now.
In order to avoid sending/receiving ID, use option --noid
Sending: 4 ID ("name" "imapsync" "version" "2.200" "os" "darwin" "vendor" "Gilles LAMIRAL" "support-url" "https://imapsync.lamiral.info/" "date" "04-Apr-2022 16:05:55 +0000" "side" "host2")
Sent 182 bytes
Read: 	* ID ("name" "EA" "version" "COMS/EA21.01/r20220419")
  	4 OK ID done

Host2: QUOTA capability found, presented in raw IMAP on next lines
Sending: 5 GETQUOTAROOT INBOX
Sent 22 bytes
Read: 	* QUOTAROOT "INBOX" ""
  	* QUOTA "" (STORAGE 5 20971520)
  	5 OK GETQUOTAROOT done

Host2: Quota current storage is 5120 bytes. Limit is 21474836480 bytes. So 0.00 % full
Host1: Compression is off. Use --compress1 to allow compression on host1
Host2: Compression is off. Use --compress2 to allow compression on host2
Host1: found 16 folders.
Host2: found 17 folders.
Host1: guessing separator from folder listing: [/]
Host1: separator given by NAMESPACE: [/]
Host2: guessing separator from folder listing: [/]
Host2: separator given by NAMESPACE: [/]
Host1: guessing prefix from folder listing: []
Host1: prefix given by NAMESPACE: []
Host2: guessing prefix from folder listing: []
Host2: prefix given by NAMESPACE: []
Host1: separator and prefix: [/][]
Host2: separator and prefix: [/][]
Including all folders found by default. Use --subscribed or --folder or --folderrec or --include to select specific folders. Use --exclude to unselect specific folders.
Host1: Checking wanted folders exist. Use --nocheckfoldersexist to avoid this check (shared of public namespace targeted).
Host1: Checking wanted folders are selectable. Use --nocheckselectable to avoid this check.
Turned on automapping folders ( use --noautomap to turn off automapping )
Host1: special [Gmail]/All Mail     = \All
Host1: special [Gmail]/Drafts       = \Drafts
Host1: special [Gmail]/Sent Mail    = \Sent
Host1: special [Gmail]/Spam         = \Junk
Host1: special [Gmail]/Starred      = \Flagged
Host1: special [Gmail]/Trash        = \Trash

Host2: special trash                = \Trash
Host2: special sent                 = \Sent
Host2: special draftbox             = \Drafts
Host2: special unsolbox             = \Junk

Host1: Trash not \Trash because set to Deleted Messages

++++ Listing folders
All foldernames are presented between brackets like [X] where X is the foldername.
When a foldername contains non-ASCII characters it is presented in the form
[X] = [Y] where
X is the imap foldername you have to use in command line options and
Y is the utf8 output just printed for convenience, to recognize it.

Host1: folders list (first the raw imap format then the [X] = [Y]):
* LIST (\HasNoChildren) "/" "Deleted Messages"
* LIST (\HasNoChildren) "/" "INBOX"
* LIST (\HasNoChildren) "/" "Misc"
* LIST (\HasNoChildren) "/" "Priority"
* LIST (\HasNoChildren) "/" "RecipeML"
* LIST (\HasNoChildren) "/" "Sent Messages"
* LIST (\HasNoChildren) "/" "Textpattern"
* LIST (\HasNoChildren) "/" "Trash"
* LIST (\HasChildren \Noselect) "/" "[Gmail]"
* LIST (\All \HasNoChildren) "/" "[Gmail]/All Mail"
* LIST (\Drafts \HasNoChildren) "/" "[Gmail]/Drafts"
* LIST (\HasNoChildren \Important) "/" "[Gmail]/Important"
* LIST (\HasNoChildren \Sent) "/" "[Gmail]/Sent Mail"
* LIST (\HasNoChildren \Junk) "/" "[Gmail]/Spam"
* LIST (\Flagged \HasNoChildren) "/" "[Gmail]/Starred"
* LIST (\HasNoChildren \Trash) "/" "[Gmail]/Trash"
* LIST (\HasNoChildren) "/" "iText"
26 OK Success

[Deleted Messages]
[INBOX]
[Misc]
[Priority]
[RecipeML]
[Sent Messages]
[Textpattern]
[Trash]
[[Gmail]/All Mail]
[[Gmail]/Drafts]
[[Gmail]/Important]
[[Gmail]/Sent Mail]
[[Gmail]/Spam]
[[Gmail]/Starred]
[[Gmail]/Trash]
[iText]

Host2: folders list (first the raw imap format then the [X] = [Y]):
* LIST () "/" "inbox"
* LIST (\Trash) "/" "trash"
* LIST (\Sent) "/" "sent"
* LIST (\Drafts) "/" "draftbox"
* LIST (\Junk) "/" "unsolbox"
* LIST () "/" "Misc"
* LIST () "/" "Priority"
* LIST () "/" "RecipeML"
* LIST () "/" "Textpattern"
* LIST () "/" "Important"
* LIST () "/" "Starred"
* LIST () "/" "iText"
* LIST () "/" "All Mail"
* LIST () "/" "newsletter"
* LIST () "/" "social"
* LIST () "/" "register"
* LIST () "/" "ecard"
11 OK LIST done

[All Mail]
[Important]
[Misc]
[Priority]
[RecipeML]
[Starred]
[Textpattern]
[draftbox]
[ecard]
[iText]
[inbox]
[newsletter]
[register]
[sent]
[social]
[trash]
[unsolbox]

Folders in host2 not in host1:
[social]
[register]
[newsletter]
[inbox]
[ecard]

Folders mapping from --automap feature (use --f1f2 to override any mapping):
[[Gmail]/Drafts]                         -> [draftbox]                              
[[Gmail]/Spam]                           -> [unsolbox]                              
[[Gmail]/Sent Mail]                      -> [sent]                                  
[Trash]                                  -> [trash]                                 
[Deleted Messages]                       -> [trash]                                 
[Sent Messages]                          -> [sent]                                  
[[Gmail]/Starred]                        -> [Starred]                               
[[Gmail]/Trash]                          -> [trash]                                 

Checking SEARCH ALL works on both accounts. To avoid that check, use --nochecknoabletosearch
Host1: checking if SEARCH ALL works on INBOX
Host1: folder [INBOX] has 264 messages mentioned by SELECT
Host1: folder [INBOX] has 264 messages found by SEARCH ALL
Host1: folder [INBOX] has the same messages count (264) by SELECT and SEARCH ALL
Host2: checking if SEARCH ALL works on INBOX
Host2: folder [INBOX] has 3 messages mentioned by SELECT
Host2: folder [INBOX] has 3 messages found by SEARCH ALL
Host2: folder [INBOX] has the same messages count (3) by SELECT and SEARCH ALL
Good! SEARCH ALL works on both accounts.

Folders sizes before the synchronization. It can take some time. Be patient.
You can remove foldersizes listings by using "--nofoldersizes" and "--nofoldersizesatend"
but then you will also lose the ETA (Estimation Time of Arrival) given after each message copy.
Host1 folder    1/16 [Deleted Messages]                  Size:         0 Messages:     0 Biggest:         0
Host2 folder    1/16 [trash]                             Size:         0 Messages:     0 Biggest:         0
Host2-Host1                                                            0               0                  0

Host1 folder    2/16 [INBOX]                             Size:   2945827 Messages:   264 Biggest:    185412
Host2 folder    2/16 [INBOX]                             Size:      5633 Messages:     3 Biggest:      4201
Host2-Host1                                                     -2940194            -261            -181211

Host1 folder    3/16 [Misc]                              Size:         0 Messages:     0 Biggest:         0
Host2 folder    3/16 [Misc]                              Size:         0 Messages:     0 Biggest:         0
Host2-Host1                                                            0               0                  0

Host1 folder    4/16 [Priority]                          Size:         0 Messages:     0 Biggest:         0
Host2 folder    4/16 [Priority]                          Size:         0 Messages:     0 Biggest:         0
Host2-Host1                                                            0               0                  0

Host1 folder    5/16 [RecipeML]                          Size:      4280 Messages:     1 Biggest:      4280
Host2 folder    5/16 [RecipeML]                          Size:         0 Messages:     0 Biggest:         0
Host2-Host1                                                        -4280              -1              -4280

Host1 folder    6/16 [Sent Messages]                     Size:      4922 Messages:     1 Biggest:      4922
Host2 folder    6/16 [sent]                              Size:         0 Messages:     0 Biggest:         0
Host2-Host1                                                        -4922              -1              -4922

Host1 folder    7/16 [Textpattern]                       Size:         0 Messages:     0 Biggest:         0
Host2 folder    7/16 [Textpattern]                       Size:         0 Messages:     0 Biggest:         0
Host2-Host1                                                            0               0                  0

Host1 folder    8/16 [Trash]                             Size:         0 Messages:     0 Biggest:         0
Host2 folder    8/16 [trash]                             Size:         0 Messages:     0 Biggest:         0
Host2-Host1                                                            0               0                  0

Host1 folder    9/16 [[Gmail]/Drafts]                    Size:         0 Messages:     0 Biggest:         0
Host2 folder    9/16 [draftbox]                          Size:         0 Messages:     0 Biggest:         0
Host2-Host1                                                            0               0                  0

Host1 folder   10/16 [[Gmail]/Important]                 Size:   2498263 Messages:   196 Biggest:    185412
Host2 folder   10/16 [Important]                         Size:         0 Messages:     0 Biggest:         0
Host2-Host1                                                     -2498263            -196            -185412

Host1 folder   11/16 [[Gmail]/Sent Mail]                 Size:   6646884 Messages:   188 Biggest:   6039635
Host2 folder   11/16 [sent]                              Size:         0 Messages:     0 Biggest:         0
Host2-Host1                                                     -6646884            -188           -6039635

Host1 folder   12/16 [[Gmail]/Spam]                      Size:    664270 Messages:    29 Biggest:     94545
Host2 folder   12/16 [unsolbox]                          Size:         0 Messages:     0 Biggest:         0
Host2-Host1                                                      -664270             -29             -94545

Host1 folder   13/16 [[Gmail]/Starred]                   Size:      5274 Messages:     1 Biggest:      5274
Host2 folder   13/16 [Starred]                           Size:         0 Messages:     0 Biggest:         0
Host2-Host1                                                        -5274              -1              -5274

Host1 folder   14/16 [[Gmail]/Trash]                     Size:   1373364 Messages:    29 Biggest:    170134
Host2 folder   14/16 [trash]                             Size:         0 Messages:     0 Biggest:         0
Host2-Host1                                                     -1373364             -29            -170134

Host1 folder   15/16 [iText]                             Size:         0 Messages:     0 Biggest:         0
Host2 folder   15/16 [iText]                             Size:         0 Messages:     0 Biggest:         0
Host2-Host1                                                            0               0                  0

Host1 folder   16/16 [[Gmail]/All Mail]                  Size:  15948827 Messages:   871 Biggest:   6039635
Host2 folder   16/16 [All Mail]                          Size:         0 Messages:     0 Biggest:         0
Host2-Host1                                                    -15948827            -871           -6039635

Host1 Nb folders:                    16 folders
Host2 Nb folders:                    13 folders

Host1 Nb messages:                 1580 messages
Host2 Nb messages:                    3 messages

Host1 Total size:              30091911 bytes (28.698 MiB)
Host2 Total size:                  5633 bytes (5.501 KiB)

Host1 Biggest message:          6039635 bytes (5.760 MiB)
Host2 Biggest message:             4201 bytes (4.103 KiB)

Time spent on sizing:        41.5 seconds
++++ Looping on each one of 16 folders to sync
ETA: Saturday 04 June 2022-06-04 08:06:01 -0400 EDT  0 s  1580/1580 msgs left
Folder    1/16 [Deleted Messages]                  -> [trash]                            
Host1: folder [Deleted Messages] has 0 messages in total (mentioned by SELECT)
Host2: folder [trash] has 0 messages in total (mentioned by SELECT)
Host2: folder [trash] permanentflags: 
Host1: folder [Deleted Messages] considering 0 messages
Host2: folder [trash] considering 0 messages
Host1: folder [Deleted Messages] selected 0 messages, duplicates 0
Host2: folder [trash] selected 0 messages, duplicates 0
ETA: Saturday 04 June 2022-06-04 08:36:20 -0400 EDT  1817 s  1580/1580 msgs left
Folder    2/16 [INBOX]                             -> [INBOX]                            
Host1: folder [INBOX] has 264 messages in total (mentioned by SELECT)
Creating folder [INBOX] on host2
Folder [INBOX] already exists
Host2: folder [INBOX] has 3 messages in total (mentioned by SELECT)
Host2: folder [INBOX] permanentflags: 
Host2: Subscribing to folder INBOX
Host1: folder [INBOX] considering 264 messages
Host2: folder [INBOX] considering 3 messages
Host1: folder [INBOX] selected 264 messages, duplicates 0
Host2: folder [INBOX] selected 3 messages, duplicates 0
- msg INBOX/23 {2110} could not append ( Subject:[FileList.org user registration confirmation], Date:["22-Apr-2005 02:42:22 +0000"], Size:[2110], Flags:[$NotJunk NotJunk \Seen] ) to folder INBOX: 59 NO Temporary unavailable
- msg INBOX/27 {2125} could not append ( Subject:[ET user registration confirmation], Date:["27-Apr-2005 10:57:59 +0000"], Size:[2125], Flags:[$NotJunk NotJunk \Seen] ) to folder INBOX: 60 NO Temporary unavailable
- msg INBOX/40 {2390} could not append ( Subject:[Your EuroDNS account has been created], Date:["30-Nov-2005 15:17:56 +0000"], Size:[2390], Flags:[$NotJunk NotJunk \Seen] ) to folder INBOX: 61 NO Temporary unavailable
[...]
- msg INBOX/6721 {2796} could not append ( Subject:[...], Date:["13-Feb-2015 16:40:31 +0000"], Size:[2796], Flags:[\Seen] ) to folder INBOX: 98 NO Temporary unavailable
- msg INBOX/6730 {3590} could not append ( Subject:[Hover confirmation for order # 74122d8a9500289d2123], Date:["14-Feb-2015 11:07:59 +0000"], Size:[3590], Flags:[\Seen] ) to folder INBOX: 99 NO Temporary unavailable
- msg INBOX/6813 {2506} could not append ( Subject:[ICANN Domain Name Courtesy Notice 2015-04-10], Date:["12-Mar-2015 05:29:42 +0000"], Size:[2506], Flags:[\Seen] ) to folder INBOX: 100 NO Temporary unavailable
- msg INBOX/6860 {3685} could not append ( Subject:[You have Hover services renewing in 15 days], Date:["25-Mar-2015 06:58:51 +0000"], Size:[3685], Flags:[\Seen] ) to folder INBOX: 101 NO Temporary unavailable
- msg INBOX/6895 {3680} could not append ( Subject:[You have Hover services renewing in 7 days], Date:["02-Apr-2015 07:27:59 +0000"], Size:[3680], Flags:[\Seen] ) to folder INBOX: 102 NO Temporary unavailable
- msg INBOX/6910 {2507} could not append ( Subject:[ICANN Domain Name Courtesy Notice 2015-04-10], Date:["06-Apr-2015 05:29:06 +0000"], Size:[2507], Flags:[\Seen] ) to folder INBOX: 103 NO Temporary unavailable
Maximum number of errors 50 reached ( you can change 50 to any value, for example 100 with --errorsmax 100 ). Exiting.
++++ Listing 50 errors encountered during the sync ( avoid this listing with --noerrorsdump ).
Err 6/50: - msg INBOX/23 {2110} could not append ( Subject:[FileList.org user registration confirmation], Date:["22-Apr-2005 02:42:22 +0000"], Size:[2110], Flags:[$NotJunk NotJunk \Seen] ) to folder INBOX: 59 NO Temporary unavailable
Err 7/50: - msg INBOX/27 {2125} could not append ( Subject:[ET user registration confirmation], Date:["27-Apr-2005 10:57:59 +0000"], Size:[2125], Flags:[$NotJunk NotJunk \Seen] ) to folder INBOX: 60 NO Temporary unavailable
Err 8/50: - msg INBOX/40 {2390} could not append ( Subject:[Your EuroDNS account has been created], Date:["30-Nov-2005 15:17:56 +0000"], Size:[2390], Flags:[$NotJunk NotJunk \Seen] ) to folder INBOX: 61 NO Temporary unavailable
[...]
Err 45/50: - msg INBOX/6721 {2796} could not append ( Subject:[...], Date:["13-Feb-2015 16:40:31 +0000"], Size:[2796], Flags:[\Seen] ) to folder INBOX: 98 NO Temporary unavailable
Err 46/50: - msg INBOX/6730 {3590} could not append ( Subject:[Hover confirmation for order # 74122d8a9500289d2123], Date:["14-Feb-2015 11:07:59 +0000"], Size:[3590], Flags:[\Seen] ) to folder INBOX: 99 NO Temporary unavailable
Err 47/50: - msg INBOX/6813 {2506} could not append ( Subject:[ICANN Domain Name Courtesy Notice 2015-04-10], Date:["12-Mar-2015 05:29:42 +0000"], Size:[2506], Flags:[\Seen] ) to folder INBOX: 100 NO Temporary unavailable
Err 48/50: - msg INBOX/6860 {3685} could not append ( Subject:[You have Hover services renewing in 15 days], Date:["25-Mar-2015 06:58:51 +0000"], Size:[3685], Flags:[\Seen] ) to folder INBOX: 101 NO Temporary unavailable
Err 49/50: - msg INBOX/6895 {3680} could not append ( Subject:[You have Hover services renewing in 7 days], Date:["02-Apr-2015 07:27:59 +0000"], Size:[3680], Flags:[\Seen] ) to folder INBOX: 102 NO Temporary unavailable
Err 50/50: - msg INBOX/6910 {2507} could not append ( Subject:[ICANN Domain Name Courtesy Notice 2015-04-10], Date:["06-Apr-2015 05:29:06 +0000"], Size:[2507], Flags:[\Seen] ) to folder INBOX: 103 NO Temporary unavailable
The most frequent error is ERR_APPEND. 
Maximum number of errors 50 reached ( you can change 50 to any value, for example 100 with --errorsmax 100 ). Exiting.
Exiting with return value 114 (EXIT_ERR_APPEND) 50/50 nb_errors/max_errors PID 12414
Removing pidfile ./tmp/imapsync.pid
Disconnecting from host1 imap.gmail.com user1 junk@...
Disconnecting from host2 mail.mailo.com user2 junk@...
Log file is LOG_imapsync/2022_06_04_08_05_20_126_junk@..._junk@....txt ( to change it, use --logfile filepath ; or use --nolog to turn off logging )
```

**Comment by vagari** [comment]
Well I feel silly. I still have no clue what the issue is, but I tried "manually" copying between other folders in Postbox, like sent:sent, or SPAM:SPAM, or Inbox to any generated folder, and it worked. 

So I reran `imapsync` with `--exclude 'INBOX'` and it worked. I'll have to double check I have everything but I assume anything that _would_ be in the INBOX would also be in "All Mail" so it should have come over. 

I'm still curious if anyone has any insights. I still have 3 other accounts to migrate, and one is pretty huge (what is used to sign up for almost every major service). So at least I have a workaround. but I _am_ curious. Thanks!

**Comment by gilleslamiral** [comment]
I suspect something on the MAILO side since the message "NO. Temporary unavailable" comes from it.
Can you manually copy a message to INBOX at this MAILO account?

**Comment by vagari** [comment]
Yes, once I had messages copied over, in "All Mail" for example, I was able to copy them to the Inbox. 

I actually did a little more messing around last night since I noticed that I was missing some emails. What was happening in this situation was "Sent" was going before other things and I regularly send emails to myself as a reminder. Thus they weren't showing up anywhere else, even "All Mail," because they were in "Sent." 

That led me down a path of a) fiddling with the Gmail settings (like disabling the inbox categories, "Promotions, Social, etc." and b) switching to the "manual" options declaration instead of `--gmail1`. The latter sorted out the missing emails I think by putting "Sent" at the end. But while doing that I also removed the `--exclude "INBOX"`, out of curiosity, and reran it... wouldn't ya know, the Inbox started copying over. 

I've got more inboxes to move, so I'll see if the next one is as problematic. But I _think_ I have the bulk of the first migrated.

**Comment by vagari** [comment]
Second account had same issue. Doing similar things to first time didn't resolve it. I've submitted a ticket to mailo support. Fingers crossed they have a solution, I'll report back.

**Comment by gilleslamiral** [comment]
He doesn't look related to Gmail or Imapsync.
It looks related to what happens after the creation of the new mailbox.

**Comment by vagari** [comment]
It was definitely an odd thing. It happened on all four accounts I transferred. After a few days, the inbox would "magically" just start to work. It'd be nice to know what caused it on their end. But at least I got everything moved over... Just in time to see Google reversed course and left an option for personal use. Not like that matters, I'd rather pay a small fee and keep them away from at least some of my data. 

Thanks for your help.
